src/codewhisperer/util/supplementalContext/utgUtils.ts

```diff
@@ -238,15 +238,7 @@ async function findSourceFileByName(
 
 // TODO: Replace this by vscode.workspace.findFiles
 function globPromise(pattern: string): Promise<string[]> {
-    return new Promise<string[]>((resolve, reject) => {
-        glob(pattern, (err, files) => {
-            if (err) {
-                reject(err)
-            } else {
-                resolve(files)
-            }
-        })
-    })
+    return glob(pattern)
```

Comment:
When I ran the tests locally this code was not hit. `isUtg` is never `true`? https://github.com/aws/aws-toolkit-vscode/blob/0d780ccd2ae34e06464e3c740bdb61d280e6e86a/src/codewhisperer/util/supplementalContext/supplementalContextUtil.ts#L35-L36

@Will-ShaoHua @leigaol

Reply:
@justinmk3 this will return true if the file is under `/test, /tst, /tests` or following specific test file naming convention like `JavaFileTest.java` etc

Reply:
> When I ran the tests locally this code was not hit
think we might miss test cases for this path ?

Reply:
this is the test case for `isTestFile` 
https://github.com/aws/aws-toolkit-vscode/blob/master/src/test/codewhisperer/util/codeParsingUtil.test.ts#L65

Reply:
> for `isTestFile`. They list some scenarios where isUtg = true
> https://github.com/aws/aws-toolkit-vscode/blob/master/src/test/codewhisperer/util/codeParsingUtil.test.ts#L65

Ok, but none of those tests invoke `fetchSupplementalContext`.

Reply:
yea I think that's the problem from our end.